Source/Initialization/WarpXAMReXInit.cpp

```diff
@@ -44,6 +47,22 @@ namespace {
             pp_amr.add("blocking_factor", 1);
         }
 
+        //See https://github.com/AMReX-Codes/amrex/pull/3763
+#ifdef AMREX_USE_GPU
+        bool warpx_do_device_synchronize = true;
+#else
+        bool warpx_do_device_synchronize = false;
+#endif
+        pp_warpx.query("do_device_synchronize", warpx_do_device_synchronize);
+        bool do_device_synchronize = warpx_do_device_synchronize;
+        amrex::ParmParse pp_tiny_profiler("tiny_profiler");
+        if (pp_tiny_profiler.queryAdd("device_synchronize_around_region", do_device_synchronize) )
```

Comment:
If I understand this correctly, this would error out every time someone wants to run a simulation using a used input parameters file.